README.md

```diff
@@ -6,33 +6,103 @@ This plugin allows you to mount remote folder using sshfs in your container easi
 
 ## Usage
 
+### Use with SSH keys
 1 - Install the plugin
 
 ```
 $ docker plugin install vieux/sshfs # or docker plugin install vieux/sshfs DEBUG=1
+Plugin "vieux/sshfs" is requesting the following privileges:
+ - network: [host]
+ - mount: [/tmp]
+ - device: [/dev/fuse]
+ - capabilities: [CAP_SYS_ADMIN]
+Do you grant the above permissions? [y/N] y
+latest: Pulling from vieux/sshfs
+e248a6530152: Download complete
+Status: Downloaded newer image for vieux/sshfs:latest
+Installed plugin vieux/sshfs
+```
+
+2 - Configure driver to point to your SSH keys
+
+_NOTE_: The plugin defaults to looking for SSH keys in `/tmp`. You can copy your `<identity_file_name>` to `/tmp` and omit this step, e.g.
```

Comment:
@stevvooe Hm... I don't know enough about what is provided to plugins to know whether we can find that out - is it possible from within a container to find the docker user, without mounting the docker directory in or some such?

Otherwise, the only thing I can think of is `/home`?  :|

Also, the user can manually change the mounted directory later to their actual home directory.